echo.c

```diff
@@ -404,6 +404,13 @@ static void
 AddMessageBoxText(HWND hwnd, const wchar_t *text, const wchar_t *title, const wchar_t *from)
 {
     HWND hmsg = GetDlgItem(hwnd, ID_TXT_MESSAGE);
+    DWORD align[2] = {PFA_LEFT, PFA_RIGHT}; /* default alignments for text and title */
+
+    if (LangFlowDirection() == 1)
```

Comment:
I wonder why Windows hasn't provided macros for reading layout [values](https://learn.microsoft.com/en-us/windows/win32/intl/locale-ireadinglayout).